src/utils/proxy.mjs

```diff
@@ -363,6 +364,11 @@ const initializeProxy = async function ({ configPath, distDir, host, port, proje
     }
   })
   proxy.on('proxyRes', (proxyRes, req, res) => {
+    if (proxyRes.headers[edgeFunctionHeaders.EdgeFunctionBypass]) {
+      req.headers[edgeFunctionHeaders.Passthrough] = 'true'
+      return proxy.web(req, res, { target: entrypoint })
```

Comment:
this feels very hacky, as it basically makes a full second request to the dev server. I'd prefer if we could continue the existing request chain in some way, but I don't see how.